bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/DY_ZprimeToZH_ZbbHtautau/Zprime_Zh_Zbbhtautau_narrow_template/Zprime_Zh_Zbbhtautau_narrow_customizecards.dat

```diff
@@ -0,0 +1,5 @@
+set param_card mass 9000001 MASS
+set param_card mass 9000002 MASS
+set param_card rhoinputs 2 MASS
+set param_card DECAY 9000001 1.000000e-03
```

Comment:
one more question, will you use the cross section from the sample. if you use a random small number for the width, the final cross section will be incorrect. but it's fine you only need the signal shape